src/logic/safe/store/actions/processTransaction.ts

```diff
@@ -192,25 +190,7 @@ export const processTransaction =
         dispatch(updateTransactionStatus({ safeTxHash: tx.safeTxHash, status: LocalTransactionStatus.PENDING_FAILED }))
       }
 
-      const executeData = safeInstance.methods.approveHash(txHash || '').encodeABI()
-      const contractErrorMessage = await getContractErrorMessage({
-        safeInstance,
-        from,
-        data: executeData,
-      })
-
-      if (contractErrorMessage) {
-        logError(Errors._804, contractErrorMessage)
-      }
-
-      const notification = isTxPendingError(err)
-        ? NOTIFICATIONS.TX_PENDING_MSG
-        : {
-            ...notificationsQueue.afterExecutionError,
-            ...(contractErrorMessage && {
-              message: `${notificationsQueue.afterExecutionError.message} - ${contractErrorMessage}`,
-            }),
-          }
+      const notification = NOTIFICATIONS.TX_PENDING_MSG
```

Comment:
This seems to do the job.
Just wondering if the notification (after rejecting executing tx) is accurate.
<img width="361" alt="Screen Shot 2022-01-10 at 09 50 40" src="https://user-images.githubusercontent.com/32431609/148739308-9b1157e4-86d7-440e-9da4-e49482379888.png">

Reply:
This was the 'default' notification before contract errors were retrieved.